bedrock/firefox/templates/firefox/new/basic/base_download.html

```diff
@@ -59,14 +60,13 @@
   <div class="mzp-c-notification-bar mzp-t-warning" data-nosnippet="true">{{ ftl('firefox-new-mozilla-no-longer-provides', fallback='firefox-new-youre-using-an-insecure-outdated', url='https://support.mozilla.org/kb/end-support-windows-xp-and-vista') }}</div>
   <![endif]-->
 
-  {% call hero(
-    title=hero_title,
-    class='mzp-has-image hero-firefox t-narrow',
-    include_cta=True,
-    image_url=hero_img,
-    heading_level=1
-  ) %}
+  {% call split(
```

Comment:
**issue (blocking)**: should keep CTA above image on mobile
<img width="705" alt="download-split" src="https://user-images.githubusercontent.com/19650432/212359684-df8d713a-9c93-48df-ba40-f88323e52eb5.png">

**suggestion**: add media after True param